### ISSUE TITLE
Can apptainer open multi ports?

### ISSUE THREAD

**Issue by xigyou** [apptainer/apptainer#701]
### Version of Apptainer
What version of Apptainer (or Singularity) are you using? Run
`apptainer --version` (or `singularity --version`).
1.0.1

### Expected behavior
open the 8929 port and 2289 port .

What did you expect to see when you do...?
--net --network-args "portmap=8929:8929/tcp" --network-args "portmap=2289:22/tcp"

### Actual behavior
only the 8929 port is work, 2289 is not work.

What actually happend? Why was it incorrect?

### Steps to reproduce this behavior

How can others reproduce this issue/problem?

And the  --network bridge also get the IP addr I want
hostname -I
192.168.8.4 10.22.0.1
Apptainer> hostname -I
10.22.0.13 

When I  create a .conflist file in /usr/local/etc/apptainer/network/，it's also error.
cat /usr/local/etc/apptainer/network/00_bridge8.conflist
{
    "cniVersion": "1.0.0",
    "name": "bridge8",
    "plugins": [
        {
            "type": "bridge",
            "bridge": "ens16f0",
            "isGateway": true,
            "ipMasq": true,
            "ipam": {
                "type": "host-local",
                "subnet": "192.168.8.0/16",
                "routes": [
                    { "dst": "0.0.0.0/0" }
                ]
            }
        },
        {
            "type": "firewall"
        },
        {
            "type": "portmap",
            "capabilities": {"portMappings": true},
            "snat": true
        }
    ]
}

 failed to create bridge "ens16f0": "ens16f0" already exists but is not a bridge

### What OS/distro are you running

```sh
$ cat /etc/os-release
cat /etc/os-release
NAME="CentOS Stream"
VERSION="9"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="9"
PLATFORM_ID="platform:el9"
PRETTY_NAME="CentOS Stream 9"
ANSI_COLOR="0;31"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:centos:centos:9"
HOME_URL="https://centos.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux 9"
REDHAT_SUPPORT_PRODUCT_VERSION="CentOS Stream"

```

### How did you install Apptainer
make install 
Write here how you installed Apptainer (or Singularity). Eg. RPM, source.
forgot.

**Comment by cclerget** [comment]
@xigyou Yes apptainer support multiple ports, looks like you have an error in your network configuration, the `bridge` name can be anything except an existing interface/bridge name like you here (`ens16f0`), so `00_bridge8.conflist` would look like this :

```
{
    "cniVersion": "1.0.0",
    "name": "bridge8",
    "plugins": [
        {
            "type": "bridge",
            "bridge": "bridge8",
            "isGateway": true,
            "ipMasq": true,
            "ipam": {
                "type": "host-local",
                "subnet": "192.168.8.0/16",
                "routes": [
                    { "dst": "0.0.0.0/0" }
                ]
            }
        },
        {
            "type": "firewall"
        },
        {
            "type": "portmap",
            "capabilities": {"portMappings": true},
            "snat": true
        }
    ]
}
```

And to use this bridge, you have to specify it with `--network` parameter like this :

```
--net --network bridge8 --network-args "portmap=8929:8929/tcp" --network-args "portmap=2289:22/tcp"
```
